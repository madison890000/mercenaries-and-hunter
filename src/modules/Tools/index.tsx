import Button from "@mui/material/Button";
import { useNavigate } from "react-router";
import { useIntl } from "react-intl";


const Tools = () => {
  const navigate = useNavigate();
  const intl = useIntl();
  return (
    <div style={{
      padding: 30,
      display: "flex",
      justifyContent: 'space-between'
    }}>
      <Button variant="contained" size="large" onClick={() => {
        navigate("/tools/translate")
      }} style={{
        backgroundColor: '#6096B4',
        padding:40,
      }}>{intl.formatMessage({ id: 'menu.format-and-translate' })}</Button>
      <Button variant="contained" size="large" onClick={() => {
        navigate("/tools/cl")
      }} style={{
        backgroundColor: '#6096B4',
        padding:40,
      }}>{intl.formatMessage({ id: 'menu.auto-cl' })}</Button>
      <Button variant="contained" size="large" onClick={() => {
        navigate("/tools/import")
      }} style={{
        backgroundColor: '#6096B4',
        padding:40,
      }}>{intl.formatMessage({ id: 'menu.import' })}</Button>
    </div>
  )
}

export default Tools