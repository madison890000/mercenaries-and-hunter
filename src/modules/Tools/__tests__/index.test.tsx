import React, {PropsWithChildren} from 'react';
import Tools from '../index';
import {render} from '@testing-library/react';
import {BrowserRouter as Router} from 'react-router-dom';

const RouterWrapper: React.FC<PropsWithChildren> = ({children}) => {
    return <Router>{children}</Router>;
};
test('renders Tools', () => {
    const {asFragment} = render(
        <RouterWrapper>
            <Tools/>
        </RouterWrapper>
    );
    expect(asFragment()).toMatchSnapshot();
});
