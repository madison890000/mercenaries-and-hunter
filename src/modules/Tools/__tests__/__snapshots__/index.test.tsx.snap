// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Tools 1`] = `
<DocumentFragment>
  <div
    style="padding: 30px; display: flex; justify-content: space-between;"
  >
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeLarge MuiButton-containedSizeLarge MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeLarge MuiButton-containedSizeLarge css-jh47zj-MuiButtonBase-root-MuiButton-root"
      style="background-color: rgb(96, 150, 180); padding: 40px;"
      tabindex="0"
      type="button"
    >
      AI translate
      <span
        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
      />
    </button>
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeLarge MuiButton-containedSizeLarge MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeLarge MuiButton-containedSizeLarge css-jh47zj-MuiButtonBase-root-MuiButton-root"
      style="background-color: rgb(96, 150, 180); padding: 40px;"
      tabindex="0"
      type="button"
    >
      AI Cover Letter
      <span
        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
      />
    </button>
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeLarge MuiButton-containedSizeLarge MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeLarge MuiButton-containedSizeLarge css-jh47zj-MuiButtonBase-root-MuiButton-root"
      style="background-color: rgb(96, 150, 180); padding: 40px;"
      tabindex="0"
      type="button"
    >
      Score Resume
      <span
        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
      />
    </button>
  </div>
</DocumentFragment>
`;
